test/support/apps/action_view/config.ru

```diff
@@ -11,41 +11,27 @@ class TestViewApplication < Rails::Application
   config.consider_all_requests_local = true
   config.secret_key_base = 'test_key'
   config.secret_token = 'test_token'
+  config.autoload_paths = []
+  ActiveSupport::Dependencies.autoload_paths = ActiveSupport::Dependencies.autoload_paths.dup
```

Comment:
Why do we need this line?

Reply:
ActiveSupport::Dependencies.autoload_paths are being set globally and is frozen in our test environment during initialisation of our ActionControllerTest. So these lines would unfreeze the array and helps initialise the ActionViewTest app properly. Tried turning of the autoload but did not work so i am forcefully setting these(Unfreezing).
There should be no issues as ideally in a real world rails there will be one application(In our test env we have multiple apps)